src/runtime/auto-prompt-manager.js

```diff
@@ -531,8 +532,8 @@ export class AutoPromptManager {
       );
 
       if (contributionIndex > 0) {
-        actionToUse = potentialActions[0].type;
-        this.promptDisplayed_ = actionToUse;
+        actionToUse = potentialActions[0];
+        this.promptDisplayed_ = actionToUse.type;
```

Comment:
(Not actionable) FYI
One thing to think about suppression logic for frequency-based prompts is how we're going to handle it for multi-configs type (e.g., survey). We may want to distinguish each survey for suppression then we should log impression/dismissals per configid, not per type.
Just leaving a note as it's out of scope and needs more discussion with PM/UX.

Reply:
I agree, I made a note of it on b/243704873 talking about prompt dismissals too. I will add this case to the bug as well.